src/oas2/transformers/securities.ts

```diff
@@ -76,6 +76,8 @@ export function translateToSingleSecurity(security: DeepPartial<Security>) {
   return;
 }
 
-export function translateToSecurities(securities: Array<DeepPartial<Security>>): HttpSecurityScheme[] {
-  return compact(securities.map(translateToSingleSecurity));
+export function translateToSecurities(securities: Security[][]): HttpSecurityScheme[][] {
+  return securities.map((security: Array<DeepPartial<Security>>) => {
```

Comment:
```suggestion
  return securities.map((security) => {
```

Won't this be inferred?